manifests/010-prometheus-rules.yaml

```
@@ -9,14 +9,30 @@ spec:
   groups:
   - name: SamplesOperator
     rules:
+    - alert: SamplesRetriesMissingOnImagestreamImportFailing
+      expr: sum(openshift_samples_failed_imagestream_import_info) > 0 && openshift_samples_retry_imagestream_import_total <= openshift_samples_retry_imagestream_import_total offset 30m && sum(openshift_samples_invalidsecret_info) == 0
+      for: 2h
+      labels:
+        severity: warning
+      annotations:
+        message: |
+          Samples operator is detecting problems with imagestream image imports, and the periodic retries of those imports are not occurring.  Contact support.  You can look at the "openshift-samples" ClusterOperator object for details. You can also query 'openshift_samples_failed_imagestream_import_info' to see which ImageStreams do not import. Most likely there are issues with the external image registry hosting the images that need to be investigated.  If registry.redhat.io is used, then samples operator has at least determined that pull credentials exist, and that network connections to registry.redhat.io were previously successful.
+    - alert: SamplesImagestreamImportFailing
+      expr: sum(openshift_samples_failed_imagestream_import_info) > 0 && openshift_samples_retry_imagestream_import_total  > openshift_samples_retry_imagestream_import_total offset 30m && sum(openshift_samples_invalidsecret_info) == 0
+      for: 2h
+      labels:
+        severity: warning
+      annotations:
+        message: |
+          Samples operator is detecting problems with imagestream image imports.  You can look at the "openshift-samples" ClusterOperator object for details. You can also query 'openshift_samples_failed_imagestream_import_info' to see which ImageStreams do not import. Most likely there are issues with the external image registry hosting the images that need to be investigated.  If registry.redhat.io is used, then samples operator has at least determined that pull credentials exist, and that network connections to registry.redhat.io were previously successful.  Or you can consider marking samples opertaor Removed if you do not care about having sample imagestreams available.
     - alert: SamplesDegraded
       expr: openshift_samples_degraded_info == 1
       for: 2h
       labels:
         severity: warning
       annotations:
         message: |
-          Samples could not be deployed and are in Degraded status. You can look at the "openshift-samples" ClusterOperator object for details. You can also query 'openshift_samples_failed_imagestream_import_info' to see if ImageStreams do not import. The samples operator reports itself Degraded if an ImageStream import continues to fail for 2 hours.
+          Samples could not be deployed and are in Degraded status. You can look at the "openshift-samples" ClusterOperator object for details.
```

Avoid the second-person - use an imperative instead:
```suggestion
          Samples could not be deployed and the operator degraded. Review the "openshift-samples" ClusterOperator object for further details.
```

---

part of next push